src/JSInterop/Microsoft.JSInterop/src/IJSUnmarshalledRuntime.cs

```diff
@@ -27,7 +27,7 @@ public interface IJSUnmarshalledRuntime
         /// <param name="identifier">The identifier used when registering the target function.</param>
         /// <param name="arg0">The first argument.</param>
         /// <returns>The result of the function invocation.</returns>
-        TResult InvokeUnmarshalled<T0, TResult>(string identifier, T0 arg0);
+        TResult? InvokeUnmarshalled<T0, TResult>(string identifier, T0 arg0);
```

Comment:
Can arguments be null?

Reply:
It seems like `arg0` would only be nullable when `T0` is nullable; i.e. no change is needed to the annotations for this parameter.

Reply:
I suppose yes.

Reply:
```suggestion
        TResult? InvokeUnmarshalled<T0, TResult>(string identifier, T0? arg0);
```

Reply:
@pranavkm I would lean against making that change from https://github.com/dotnet/aspnetcore/pull/25261#discussion_r477641317.